tsconfig.json

```diff
@@ -1,9 +1,12 @@
 {
     "compilerOptions": {
         "target": "es5",
-        "module": "umd",
-        "jsx": "react",
+         "module": "umd",
+         "jsx": "react",
         "declaration": true,
         "sourceMap": true
-    }
+    },
+    "files": [
+        "typings/main.d.ts"
```

Comment:
According to the typings docs, if you reference like this you shouldn't need to use ```/// <reference``` in TS files. I'm unclear on wether or not this works tho. But that probably has more to do with my massive 2 days of typescript experience than anything else.